docs/guides/testing.md

```diff
@@ -153,8 +149,17 @@ afterEach(() => {
 })
 ```
 
-> **Note:** without [globals configuration](https://vitest.dev/config/#globals) enabled, you need
-> to add `import { afterEach, vi } from 'vitest'` at the top
+> **Note**: without [globals configuration](https://vitest.dev/config/#globals) enabled, you need
+> to add `import { afterEach, vi } from 'vitest'` at the top.
+
+```ts
+// __mocks__/vitest-env.d.ts
+/// <reference types="vite/client" />
+/// <reference types="vitest/globals" />
+```
+
+> **Note**: without [globals configuration](https://vitest.dev/config/#globals) enabled, you do not
```

Comment:
Double negative is confusing here. I had to read it a couple of times to get it.

```suggestion
> **Note**: with [globals configuration](https://vitest.dev/config/#globals) enabled, you need
```

but I don't know if that was the intention.